src/genie/libs/parser/ios/asr901/tests/ShowEnvironment/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,72 @@
+expected_output = {
+    "Fan": {
+        "fan1": {
+            "speed": "53",
+            "status": "Normal"
+        },
+        "fan2": {
+            "speed": "53",
+            "status": "Normal"
+        },
+        "fan3": {
+            "speed": "54",
+            "status": "Normal"
+        }
+    },
+    "PowerSupply": {
+        "0.75V": {
+            "status": "Normal",
+            "volt": "+0.760"
+        },
+        "1.05V": {
+            "status": "Normal",
+            "volt": "+1.080"
+        },
+        "1.25V": {
+            "status": "Normal",
+            "volt": "+1.250"
+        },
+        "1.2V": {
+            "status": "Normal",
+            "volt": "+1.200"
+        },
+        "1.5V": {
+            "status": "Normal",
+            "volt": "+1.510"
+        },
+        "1.8V": {
+            "status": "Normal",
+            "volt": "+1.780"
+        },
+        "12AV": {
+            "status": "Normal",
+            "volt": "+12.100"
+        },
+        "1V": {
+            "status": "Normal",
+            "volt": "+1.000"
+        },
+        "2.5V": {
+            "status": "Normal",
+            "volt": "+2.500"
+        },
+        "3.3V": {
+            "status": "Normal",
+            "volt": "+3.240"
+        },
+        "5V": {
+            "status": "Normal",
+            "volt": "+4.920"
+        }
+    },
+    "Sensor": {
+        "Board": {
+            "status": " Normal",
+            "temp": "44"
+        },
+        "Inlet": {
+            "status": " Normal",
+            "temp": "30"
+        }
```

Comment:
missing main status. and added "type". changed to "temp_cels" to make unit clear and changed to integer.
```suggestion
    "board_temp": {
        "status": "Normal"
        "type": {
            "Board": {
                "status": " Normal",
                "temp_cels": 44, # integer
            },
            "Inlet": {
                "status": " Normal",
                "temp_cels": 33, # integer
            }
```

Reply:
confirmed the change.